citest/gcp_testing/gcp_api.py

```diff
@@ -0,0 +1,76 @@
+# Copyright 2016 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+"""Helper functions for creating authenticated GCP service clients."""
+
+import logging
+import httplib2
+
+from .. import json_contract as jc
+
+from apiclient.discovery import build as discovery_build
```

Comment:
I'd do `import apiclient.discovery as discovery` and then call `discovery.build` to be clearer here.